.ci/docker/common/install_onnx.sh

```diff
@@ -33,10 +33,16 @@ pip_install onnx-weekly==1.15.0.dev20230717
 # TODO: change this when onnx-script is on testPypi
 pip_install onnxscript-preview==0.1.0.dev20230809 --no-deps
 
+# Transformers requires packages
+pip_install datasets==2.14.4
+
 # Cache the transformers model to be used later by ONNX tests. We need to run the transformers
 # package to download the model. By default, the model is cached at ~/.cache/huggingface/hub/
 IMPORT_SCRIPT_FILENAME="/tmp/onnx_import_script.py"
-as_jenkins echo 'import transformers; transformers.AutoModel.from_pretrained("sshleifer/tiny-gpt2"); transformers.AutoTokenizer.from_pretrained("sshleifer/tiny-gpt2");' > "${IMPORT_SCRIPT_FILENAME}"
+as_jenkins echo 'import transformers; transformers.AutoModel.from_pretrained("sshleifer/tiny-gpt2").to("cpu"); transformers.AutoTokenizer.from_pretrained("sshleifer/tiny-gpt2");' \
+                'transformers.AutoModel.from_pretrained("bigscience/bloom-560m").to("cpu"); transformers.AutoTokenizer.from_pretrained("bigscience/bloom-560m");' \
+                'transformers.AutoModel.from_pretrained("openai/whisper-tiny").to("cpu"); transformers.WhisperConfig.from_pretrained("openai/whisper-tiny");transformers.WhisperProcessor.from_pretrained("openai/whisper-tiny");' \
+                'transformers.AutoModel.from_pretrained("google/flan-t5-small").to("cpu"); transformers.AutoTokenizer.from_pretrained("google/flan-t5-small");' > "${IMPORT_SCRIPT_FILENAME}"
```

Comment:
@huydhn Do you maybe know why caching more models crashes the other builds with `ImportError: /opt/conda/envs/py_3.8/lib/libstdc++.so.6: version `GLIBCXX_3.4.30' not found (required by /opt/conda/envs/py_3.8/lib/python3.8/site-packages/torch/lib/libtorch_python.so)`?